draft-claise-semver.md

```diff
@@ -147,7 +145,7 @@ RFC-formatted document as necessary.
 ~~~~~
 * a9d7d29 (tag:2.0.0, tag:RFCXXXX) Merge branch 'v2'
```

Comment:
The first two are just bugs.  

On the third point, the thing I meant to say but forgot to was: "Not every commit needs a version number."  If beta versions are like I-Ds, commits without versions are like the commits in between I-Ds.